plugins/slack/app/models/slack_notification.rb

```diff
@@ -4,18 +4,23 @@ def initialize(deploy)
     @stage = deploy.stage
     @project = @stage.project
     @user = @deploy.user
+    @webhook = @stage.slack_webhooks.first
   end
 
+
   def deliver
-    Slack.chat_postMessage(
-      token: ENV['SLACK_TOKEN'],
-      channel: @stage.slack_channels.first.channel_id,
-      text: content)
+    payload = {text: content, username: "samson-bot"}
+
+    if !@webhook.channel.blank?
+      payload.merge!({channel: @webhook.channel})
+    end
```

Comment:
@henders sure :+1: I changed it to `unless`.
I used negative-if because `unless` is a bit off to read. I have nothing against `unless`. I would gladly welcome `unless` to my house for dinner 😊